source/text/Format.cpp

```diff
@@ -89,6 +89,32 @@ string Format::Credits(int64_t value)
 
 
 
+// Convert a time in seconds to years/days/hours/minutes/seconds
+std::string Format::PlayTime(int time)
+{
+	string result;
+	static const vector<char> SUFFIX = {'s', 'm', 'h', 'd', 'y'};
+	static const vector<int> PERIOD = {60, 60, 24, 365};
```

Comment:
We should probably use `std::array` instead of `std::vector` here.
(We can clean up the other similar instances in this file in a follow-up PR.)